### DISCUSSION TITLE
How to monitor WiFi signal strength?

### DISCUSSION THREAD

**Discussion by dhoineProg** [AvaloniaUI/Avalonia#16773] [Q&A]
I have a project where I need to monitor WiFi signal strength. How can I do this in avalonia and is it even possible to do?

**Comment by superware** [comment]
"Building a UI application" and "monitoring WiFi signal strength" are two completely separate concerns.

I suggest you start by implementing a simple WiFi monitoring Console Application, for example by printing to the console the signal strength every second. Search the Web and you'll find many examples/approaches.

Later you will be able to use that code in your UI app.
